babel-polyfill/build.boot

```diff
@@ -0,0 +1,28 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "6.20.0")
+(def +version+ (str +lib-version+ "-1"))
+
+(task-options!
+  pom {:project     'cljsjs/babel-polyfill
+       :version     +version+
+       :description "Provides polyfills necessary for a full ES2015+ environment."
+       :url         "https://github.com/babel/babel/tree/master/packages/babel-polyfill"
+       :scm         {:url "https://github.com/SMX-LTD/cljsjs-packages"}
+       :license     {"MIT" "https://opensource.org/licenses/MIT"}})
+
+(deftask package []
+  (comp
+    (download :url "https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.20.0/polyfill.js")
```

Comment:
The CDN has versioned urls: https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.20.0/polyfill.js